docs/rest/README.md

```diff
@@ -0,0 +1,19 @@
+# REST
+
+The REST documentation consists of two components.
+
+## schemata
+
+[JSON Schema](http://json-schema.org/) is a vocabulary that allows you to annotate and validate JSON documents.
+
+The following REST route definitions exist:
+
+* schemata/pages.json
+* schemata/event_pages.json
+* schemata/languages.json
+* schemata/locations.json
+
+## generator
+
+The generator allows you to generate a nice html representation of your JSON Schema files.
+This implementation is likely to changes as it's not perfectly stable!
```

Comment:
should mention that other routes like `de/wp-json/extensions/v0/modified_content/posts_and_pages` also exist